csharp/test/Ice/adapterDeactivation/AllTests.cs

```diff
@@ -316,14 +316,22 @@ static EndpointInfo getUnderlying(EndpointInfo endpointInfo) =>
 
                 output.Write("testing whether server is gone... ");
                 output.Flush();
-                try
+                if (obj.ice_getConnection() is null) // collocated
                 {
-                    obj.ice_invocationTimeout(100).ice_ping(); // Use timeout to speed up testing on Windows
-                    test(false);
+                    obj.ice_ping();
```

Comment:
In the collocated case, it still works fine after deactivate. Same for bidir (not tested).